packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -736,13 +736,13 @@ export class DocumentDeltaConnection
 	}
 
 	private createErrorObjectWithProps(
-		handler: string,
+		socketEvent: string,
```

Comment:
This is not necessarily a socket event. For example, in case of "orderingServiceHandshakeTimeout". Maybe rename  it to "socketHandler"

Reply:
@jatgarg , `Handler` does not seem to be the right choice of words here, since even the event `orderingServiceHandshakeTimeout` seems to be a custom error name, nothing points to "handler". 

99% of the cases are logging the socket event name, with error as the second property to the `createErrorObject` method, which makes me think maybe we should revisit the `orderingServiceHandshakeTimeout` code logic. I can do that, however you will need to walk me through the original intent behind this error. Maybe we should do something like `createErrorObject("connect_timeout", {message: "orderingServiceHandshakeTimeout"})`